### DISCUSSION TITLE
Two more products using the vscode-amiga-debug framework

### DISCUSSION THREAD

**Discussion by abujdoso** [BartmanAbyss/vscode-amiga-debug#258] [Show and tell]
Hi Bartman,

First of all, many thanks for the environment! It really makes Amiga development far easier. 
I am happy to report that there are two more productions made with your extension:

https://www.pouet.net/prod.php?which=94749
https://www.pouet.net/prod.php?which=95887

First one is cobbled together by yours truly and I gave some pointers to the author of the second.

Thanks again,
Artur

**Comment by AdamKeher** [comment]
cool productions!

Here's another two:

https://www.pouet.net/prod.php?which=99161
https://www.pouet.net/prod.php?which=105288
